<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowserChat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>441</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSend">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listViewPlayers">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>231</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRefreshPlayers">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">enabled:false</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Players:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Chat:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rooms on the server:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNewGame">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QListView" name="listViewGames">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>211</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>270</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSoubor">
    <property name="title">
     <string>Soubor</string>
    </property>
    <addaction name="actionOdhlasit"/>
    <addaction name="actionUkoncit"/>
   </widget>
   <addaction name="menuSoubor"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOdhlasit">
   <property name="text">
    <string>Odhlásit</string>
   </property>
  </action>
  <action name="actionUkoncit">
   <property name="text">
    <string>Ukončit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
